.github/workflows/docker-image-amd64.yml

```
@@ -1,54 +1,40 @@
-          tags: ${{ steps.meta.outputs.tags }}
-          labels: ${{ steps.meta.outputs.labels }}
\ No newline at end of file
+      - name: Build and Push SGLang Image
+        run: |
+          DATETIME=$(date '+%Y%m%d_%H%M')
+          ORG=$(echo "${{ inputs.remote_repo }}" | awk -F'/' '{print $(NF-1)}' | tr '[:upper:]' '[:lower:]' | sed 's/[^a-z0-9]/-/g')
+          BRANCH=$(echo "${{ inputs.git_branch }}" | tr '[:upper:]' '[:lower:]' | sed 's/[^a-z0-9]/-/g')
+          VERSION="${DATETIME}_${ORG}_${BRANCH}"
+          git clone ${{ inputs.remote_repo }} && cd new-api
+          docker build . -t furion-sh.tencentcloudcr.com/furion/new-api:${VERSION}
+          docker push furion-sh.tencentcloudcr.com/furion/new-api:${VERSION}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)